### PR TITLE
CATROID-1067 REFACTOR ActionPhysicsFactory to Kotlin

### PR DESCRIPTION
**Pull request by frit96**
https://jira.catrob.at/browse/CATROID-1067
Refactoring of ActionPhysicsFactory to Kotlin

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/src/main/java/org/catrobat/catroid/physics/content/ActionPhysicsFactory.kt [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        val action = Actions.action(GlideToPhysicsAction::class.java)
+        action.setPosition(x, y)
+        action.setDuration(duration)
+        val scope = Scope(ProjectManager.getInstance().currentProject, sprite, sequence)
+        action.setScope(scope)
+        action.setPhysicsLook(sprite.look as PhysicsLook)
+        return action
+    }
+
+    // PHYSICS
```
Comment (adnaribo) [inline catroid/src/main/java/org/catrobat/catroid/physics/content/ActionPhysicsFactory.kt:70]: Please just delete all comments in the file

**Review by adnaribo — CHANGES_REQUESTED** [review]

File: catroid/src/main/java/org/catrobat/catroid/physics/content/ActionPhysicsFactory.kt [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+import org.catrobat.catroid.content.actions.SetVelocityAction
+import org.catrobat.catroid.content.actions.TurnLeftSpeedAction
+import org.catrobat.catroid.content.actions.TurnRightSpeedAction
+import org.catrobat.catroid.formulaeditor.Formula
+import org.catrobat.catroid.physics.PhysicsLook
+import org.catrobat.catroid.physics.PhysicsObject
+import org.catrobat.catroid.physics.PhysicsWorld
+
+class ActionPhysicsFactory : ActionFactory() {
+    private fun getPhysicsObject(sprite: Sprite): PhysicsObject = physicsWorld.getPhysicsObject(sprite)
```
Comment (blindguardian50) [inline catroid/src/main/java/org/catrobat/catroid/physics/content/ActionPhysicsFactory.kt:48]: Please move this function below the physicsWorld member.

**Review by blindguardian50 — CHANGES_REQUESTED** [review]
Good Work! Please consider my suggested change!

**Review by blindguardian50 — APPROVED** [review]
Looks good to me!

**Review by adnaribo — APPROVED** [review]
Looks good :+1:

**Pull request merged**